shared/actions/kbfs/index.platform.desktop.js

```diff
@@ -176,43 +176,50 @@ function* installFuseSaga(): SagaGenerator<any, any> {
 }
 
 function findKeybaseUninstallString(): Promise<string> {
+  console.log('findKeybaseUninstallString')
   return new Promise((resolve, reject) => {
     const regedit = require('regedit')
     const keybaseRegPath = 'HKCU\\SOFTWARE\\Keybase\\Keybase'
-    regedit.list(keybaseRegPath).on('data', function(entry) {
-      if (entry.data.values && entry.data.values.BUNDLEKEY) {
-        const uninstallRegPath =
-          'HKCU\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\' +
-          entry.data.values.BUNDLEKEY.value
-
-        regedit.list(uninstallRegPath).on('data', function(entry) {
-          if (
-            entry.data.values &&
-            entry.data.values.DisplayName &&
-            entry.data.values.DisplayName.value === 'Keybase' &&
-            entry.data.values.Publisher &&
-            entry.data.values.Publisher.value === 'Keybase, Inc.' &&
-            entry.data.values.ModifyPath &&
-            entry.data.values.BundleCachePath
-          ) {
-            if (fs.existsSync(entry.data.values.BundleCachePath.value)) {
-              var modifyPath = entry.data.values.ModifyPath.value
-              // Remove double quotes - won't work otherwise
-              modifyPath = modifyPath.replace(/"/g, '')
-              // Remove /modify and send it in with the other arguments, below
-              modifyPath = modifyPath.replace(' /modify', '')
-              resolve(modifyPath)
+    try {
```

Comment:
This just was wrapped in try/catch. I don't think it helped anything after all but it seems harmless.